<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win_plot</class>
 <widget class="QMainWindow" name="win_plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Menu">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <widget class="QwtPlot" name="qwtPlotTx">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>400</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QwtPlot" name="qwtPlotRx">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>400</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>341</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditRx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>PyQt4.Qwt5</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
